TWLight/applications/views.py

```diff
@@ -1193,23 +1202,28 @@ def post(self, request, *args, **kwargs):
 
         if batch_update_success:
             success_apps = ", ".join(map(str, batch_update_success))
-            # Translators: After a coordinator has changed the status of a number of applications, this message appears.
             messages.add_message(
                 request,
                 messages.SUCCESS,
+                # Translators: After a coordinator has changed the status of a number of applications, this message appears.
                 _("Batch update of application(s) {} successful.".format(success_apps)),
             )
         if batch_update_failed:
             failed_apps = ", ".join(map(str, batch_update_failed))
-            # Translators: After a coordinator has changed the status of a number of applications to APPROVED, if the corresponding partner(s) is/are waitlisted or has no accounts for distribution, this message appears.
             messages.add_message(
                 request,
                 messages.ERROR,
-                _(
-                    "Cannot approve application(s) {} as partner(s) with proxy authorization method is/are waitlisted and (or) has/have not enough accounts available. If not enough accounts are available, prioritise the applications and then approve applications equal to the accounts available.".format(
+                # fmt: off
+                # Translators: After a coordinator has changed the status of a number of applications to APPROVED, if the corresponding partner(s) is/are waitlisted or has no accounts for distribution, this message appears.
+                _("Cannot approve application(s) {} as partner(s) with proxy "
+                    "authorization method is/are waitlisted and (or) has/have "
+                    "not enough accounts available. If not enough accounts are "
+                    "available, prioritise the applications and then approve "
+                    "applications equal to the accounts available.".format(
                         failed_apps
                     )
                 ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string